client/src/pages/ContactsPage/ContactsPage.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import {Box, Link, Grid, Stack, Container, Typography} from "@mui/material"
+
+import './Contacts.scss'
+
+import {Form ,Social,Map} from '../../components'
+
+
+
+const ContactsPage = () => (
+    <Stack direction='column' maxWidth='1200px' justifyContent='center' m =' 50px auto 100px auto' id="contacts">
+        <Container alignItems='center' className='contacts-header'><Typography textAlign='center' variant="h4">Contact
+            Us</Typography></Container>
+        <Map/>
+        <Grid container gap='80px' justifyContent='center' mt='70px'  className='contacts__container'
+              sx={{
+                  direction: {
+                      lg: 'row',
+                      sx: 'column',
+                  },
+              }}>
+            <Grid item xs={5} minWidth='440px'>
+                <Stack direction='column' spacing={10} ml='50px' className="contacts__info">
+                    <Box>
+                        <Typography variant="h4">
+                            Contact Info
+                        </Typography>
+                        <Grid container direction='row' wrap='nowrap' pt='30px' className="contacts__credentials">
+                            <Grid container item width='70px' spacing={3} xs={2} direction='column'
+                                  className="contacts__credentials-names">
+                                <Grid item>
+                                    <Typography color='black' fontWeight='bold'>Address:</Typography>
```

Comment:
we dont have black color in our design. we use #282828  
second. this is an mui component. our theme is accessible in it, all colors are available in our theme